### ISSUE TITLE
[Request] Change CPU governor based on battery level

### ISSUE THREAD

**Issue by mizzunet** [AdnanHodzic/auto-cpufreq#315]
Currently `auto-cpufreq` changes governor according to power state (Charging and Discharging), right?

I think it would be awesome to be able to set governor based on battery levels as well. 
For example,
Charging state governor: `performance`
Discharging state governor: `schedutil`
If battery is below 40%, governor: `powersave`

Thanks

**Comment by josh1244** [comment]
I agree, this would be a good feature. Is there a way to manually control the govener?

**Comment by mizzunet** [comment]
> Is there a way to manually control the govener?

Yeah, by editing `/etc/auto-cpufreq.conf`

**Comment by AdnanHodzic** [comment]
Since I'm very limited with available time to do this myself, if you or anyone else wants to give it a try in implementing this functionality, please give it a try and [contribute to the project](https://github.com/AdnanHodzic/auto-cpufreq/#code-contribution) and you will be credited for your work as part of [future release](https://github.com/AdnanHodzic/auto-cpufreq/releases).
